common/errors.go

```diff
@@ -45,9 +45,13 @@ func (e *Errors) GetErrors() []Error {
 	return e.errors[:]
```

Comment:
We might need to do a null check here, depending on how `func (i *Issues) Errors() []common.Error` gets crafted

Reply:
I've made `*Issues` a nil-able object type so whenever the `*Issues` value is non-nil, the `*common.Errors` object will also be non-nil.